client/src/blocking.rs

```diff
@@ -138,8 +138,10 @@ impl<'a, C: Deref<Target = impl Signer> + Clone> RequestBuilder<'a, C, Box<dyn S
     }
 
     #[must_use]
-    pub fn signer<T: Signer + 'a>(mut self, signer: T) -> Self {
-        self.signers.push(Box::new(signer));
+    pub fn signers<T: Signer + Clone + 'a>(mut self, signers: &[T]) -> Self {
+        for signer in signers {
+            self.signers.push(Box::new(signer.clone()));
+        }
```

Comment:
Why add arbitrary `Clone` requirement? [It doesn't even compile](https://github.com/coral-xyz/anchor/actions/runs/10092957753/job/27910266737?pr=3116#step:14:2446) since `Keypair` doesn't implement `Clone`: 

```
error[E0277]: the trait bound `solana_sdk::signature::Keypair: Clone` is not satisfied
   --> src/blocking.rs:125:18
    |
125 |         .signers(&[dummy_a, dummy_b])
    |          ------- ^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `solana_sdk::signature::Keypair`
    |          |
    |          required by a bound introduced by this call
    |
note: required by a bound in `anchor_client::blocking::<impl anchor_client::RequestBuilder<'a, C, Box<(dyn solana_sdk::signature::Signer + 'a)>>>::signers`
```